libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/roomlist/RoomListExtensions.kt

```diff
@@ -93,7 +93,11 @@ fun RoomListService.syncIndicator(): Flow<RoomListServiceSyncIndicator> =
             }
         }
         tryOrNull {
-            syncIndicator(listener)
+            syncIndicator(
+                200u, // Same value which was previously hardcoded in the rust-sdk
```

Comment:
Can you use 1000u? we wanted to expand it a bit. Also can we name those constants?

Reply:
OFC!